perceval/components/_pauli.py

```diff
@@ -102,3 +104,47 @@ def get_pauli_gate(pauli_type: PauliType):
         return Matrix([[1, 0], [0, -1]])
     else:
         raise NotImplementedError(f"{pauli_type}")
+
+
+def prep_state_circuit_preparer(prep_state_indices: List):
```

Comment:
We're missing the doc you removed along with the _prep_n_meas_circuits.py file :
    """
    Builds preparation circuits to prepare an input photon in each of the following
    logical qubit state states: |0>,|1>,|+>,|+i> using Pauli Gates.
    :param prep_state_indices: List of 'n'(=nqubit) indices to choose one of the logical states for each qubit
    """